db/migrate/20170510080224_create_contacts.rb

```diff
@@ -0,0 +1,32 @@
+class CreateContacts < ActiveRecord::Migration[5.1]
+  def change
+    create_table :contacts do |t|
+      t.date :birthday
+      t.string :gender
+      t.string :first_name
+      t.string :last_name
+      t.string :organisation
+      t.string :kind
+      t.string :org_role
+      t.string :title         # function in organisation
+      t.string :url
+      t.string :birthplace
+      t.text :expertise
+      t.text :hobby
+      t.text :interest
```

Comment:
We don't use vcards standard, otherwise we whould extend has_vcards but we decided to use our own, so we decide what fields are going to be in.